common/presenters/table/object-to-table-head.test.js

```diff
@@ -129,4 +131,64 @@ describe('table head presenter', function() {
       expect(i18n.t.callCount).to.equal(2)
     })
   })
+  context(
+    'with isSortable, and current active sorting on the column',
```

Comment:
I think it would be nicer to use nexted contexts here. In both contexts you're saying "with isSortable" but it think it would be clearer to use something like this instead:

```javascript
context('with isSortable', function() {
  context('with current column sorted', function() {

  })

  context('with another column sorted', function() {

  })
})
```

For me, things like _and_ in a `context` description and _when_ or _if_ in an `it` description are a bit of a flag that things should be re-worked a little.

Reply:
👍